packages/react/src/components/heliopayContainer/index.tsx

```diff
@@ -213,7 +270,7 @@ const HelioPayContainer: FC<HeliopayContainerProps> = ({
                       ) {
                         setShowFormModal(true);
                       } else if (paymentDetails) {
-                        submitPayment({
+                        submitPaymentPaylink({
```

Comment:
due to too many confusions we stoped using normalizedPrice on dash, we use decimalAmount and minimalUnitAmount